src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -0,0 +1,29 @@
+package com.google.alpollo;
+
+import java.io.IOException;
+import com.google.api.gax.rpc.FixedHeaderProvider;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.LanguageServiceSettings;
+import com.google.cloud.language.v1.Sentiment;
+
+/**
+ * Helper class with methods that use the Natural Language API or use the data that comes
+ * from the API
+ */
+public class AnalysisHelper {
+  /**
+   * Based on the lyrics sent, the AI can extract the main "sentiment" of the text.
+   * This sentiment has a score, showing the overall positivity of the text, ranging from -1.0 to 1.0
+   * and a magnitude, representing how strong the sentiment is, ranging from 0.0 to +inf
+   */
+  public Sentiment getSentiment(String lyrics) throws IOException {
```

Comment:
make the method static, that way you won't have to create an AnalysisHelper object to call it

Reply:
So can I make all the methods static in this class and use them without creating an object?

Reply:
Amazing, totally forgot about it.